README.md

```diff
@@ -60,6 +60,16 @@ Install prerequisites for E2E tests
 
 Run `dotnet test test\AllTests.proj`
 
+## Build VS Tooling
+
+Prerequisites:
+- Visual Studio 2017 15.5 (or newer)
+- Visual Studio extension development features (install via Visual Studio Installer)
+
+Open a VS Developer Command Prompt
+
+Run `msbuild BlazorTooling.sln` from the solution directory.
```

Comment:
It doesn't seem useful to include instructions about how to debug the tools until there's something to actually debug, so I left it out 😆